pkg/graphql/request.go

```diff
@@ -114,9 +114,9 @@ func (r *Request) IsIntrospectionQuery() (result bool, err error) {
 		return false, report
 	}
 
-	if r.OperationName == defaultInrospectionQueryName {
-		return true, nil
-	}
+	//if r.OperationName == defaultInrospectionQueryName {
+	//	return true, nil
+	//}
```

Comment:
This change seems good to me (see comment in #278), though I think the code should be deleted instead of being commented out. Also, the `defaultInrospectionQueryName` const at the top of this file can be deleted as well.